prometheus/handler.go

```diff
@@ -135,9 +132,16 @@ func (h *Handler) ServeHTTP(res http.ResponseWriter, req *http.Request) {
 		w = zw
 	}
 
+	h.WriteStats(w)
+}
+
+func (h *Handler) WriteStats(w io.Writer) {
```

Comment:
Would you mind adding a bit of documentation to this method?

Reply:
Thanks!!
I will put some documentation tomorrow and do a bump